src/main/functions/DidSiopMetadata.ts

```diff
@@ -1,18 +1,20 @@
+import { Format } from '@sphereon/pex-models';
+
 import { SIOP, SIOPErrors } from '../types';
 import { CommonSupportedMetadata, DiscoveryMetadataPayload, RPRegistrationMetadataPayload, SubjectIdentifierType } from '../types/SIOP.types';
 
 export function assertValidMetadata(opMetadata: DiscoveryMetadataPayload, rpMetadata: RPRegistrationMetadataPayload): CommonSupportedMetadata {
-  let methods = [];
-  const credentials = supportedCredentialsFormats(rpMetadata.credential_formats_supported, opMetadata.credential_formats_supported);
-  const isDid = verifySubjectIdentifiers(rpMetadata.subject_identifiers_supported);
-  if (isDid && rpMetadata.did_methods_supported) {
-    methods = supportedDidMethods(rpMetadata.did_methods_supported, opMetadata.did_methods_supported);
-  } else if (isDid && (!rpMetadata.did_methods_supported || !rpMetadata.did_methods_supported.length)) {
-    if (opMetadata.did_methods_supported || opMetadata.did_methods_supported.length) {
-      methods = [...opMetadata.did_methods_supported];
+  let subjectSyntaxTypesSupported = [];
+  const credentials = supportedCredentialsFormats(rpMetadata.vp_formats, opMetadata.vp_formats);
```

Comment:
Was also in the original code, but we need to rename credentials with credentialFormats, as the name is highly confusing